egs/fame/v1/run.sh

```diff
@@ -0,0 +1,300 @@
+  for subtask in 3sec 10sec 30sec; do
+    for sets in eval; do
+
+      echo "Computing VAD for data/fame_${task}_${subtask}_${sets}.."
+      sid/compute_vad_decision.sh --nj 100 --cmd "$train_cmd" \
+          data/fame_${task}_${subtask}_${sets}_enroll exp/make_vad $vaddir
+      sid/compute_vad_decision.sh --nj 100 --cmd "$train_cmd" \
+          data/fame_${task}_${subtask}_${sets}_test exp/make_vad $vaddir 
+
+    done
+  done
+done
+
+for task in ageing; do
+  for subtask in 3sec 10sec 30sec; do
+    for sets in eval; do
+      for year in _1t3 _4t10 _mt10; do
+
+      echo "Computing VAD for data/fame_${task}_${subtask}_${sets}${year}.."
+      sid/compute_vad_decision.sh --nj 100 --cmd "$train_cmd" \
+          data/fame_${task}_${subtask}_${sets}${year}_enroll exp/make_vad $vaddir
+      sid/compute_vad_decision.sh --nj 100 --cmd "$train_cmd" \
+          data/fame_${task}_${subtask}_${sets}${year}_test exp/make_vad $vaddir
+      
+      done
+    done
+  done
+done
+
+
+# Train UBM and i-vector extractor
+
+echo "Training UBM and the i-vector extractor.."
+
+sid/train_diag_ubm.sh --nj 40 --cmd "$train_cmd" \
+    data/train $num_components \
+    exp/diag_ubm_$num_components
+
+sid/train_full_ubm.sh --nj 40 --remove-low-count-gaussians false \
+    --cmd "$train_cmd" data/train \
+    exp/diag_ubm_$num_components exp/full_ubm_$num_components
+
+sid/train_ivector_extractor.sh --cmd "$train_cmd" --nj 20 --num-threads 4 --num-processes 2 \
+  --ivector-dim 600 \
+  --num-iters 5 exp/full_ubm_$num_components/final.ubm data/train \
+  exp/extractor
+
+# Extract i-vectors
+
+echo "Extracting i-vectors for data/train.."
+
+sid/extract_ivectors.sh --cmd "$train_cmd" --nj 100 \
+   exp/extractor data/train \
+   exp/ivectors_train 
+
+for task in complete ageing; do
+  for subtask in 3sec 10sec 30sec; do
+    for sets in eval; do
+
+      echo "Extracting i-vectors for data/fame_${task}_${subtask}_${sets}"
+      sid/extract_ivectors.sh --cmd "$train_cmd" --nj 100 \
+         exp/extractor data/fame_${task}_${subtask}_${sets}_enroll \
+         exp/ivectors_fame_${task}_${subtask}_${sets}_enroll
+      sid/extract_ivectors.sh --cmd "$train_cmd" --nj 100 \
+         exp/extractor data/fame_${task}_${subtask}_${sets}_test \
+         exp/ivectors_fame_${task}_${subtask}_${sets}_test
+
+    done
+  done
+done
+
+for task in ageing; do
+  for subtask in 3sec 10sec 30sec; do
+    for sets in eval; do
+      for year in _1t3 _4t10 _mt10; do
+
+      echo "Extracting i-vectors for data/fame_${task}_${subtask}_${sets}${year}"
+      sid/extract_ivectors.sh --cmd "$train_cmd" --nj 100 \
+         exp/extractor data/fame_${task}_${subtask}_${sets}${year}_enroll \
+         exp/ivectors_fame_${task}_${subtask}_${sets}${year}_enroll
+      sid/extract_ivectors.sh --cmd "$train_cmd" --nj 100 \
+         exp/extractor data/fame_${task}_${subtask}_${sets}${year}_test \
+         exp/ivectors_fame_${task}_${subtask}_${sets}${year}_test 
```

Comment:
Looks like you missed an indent level here, starting at line 193.

Reply:
Fixed